src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +242,58 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| RF receive sensitivity map                   | RB1map    | In percent units (p.u.). Radio frequency (RF) receive sensitivity maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Observed signal amplitude (S0) map           | S0map     | In arbitrary units (a.u.). For a multi-echo (typically fMRI) sequence, S0 maps index the baseline signal before exponential (T2-star) signal decay. In other words: the exponential of the intercept for a linear decay model across log-transformed echos. For more information, please see, for example, [the tedana documentation](https://tedana.readthedocs.io/en/latest/approach.html#monoexponential-decay-model-fit). S0 maps are RECOMMENDED to use this suffix if derived from an ME-FMRI dataset. |
+| Equilibrium magnetization (M0) map           | M0map     | In arbitrary units (a.u.). A common quantitative MRI (qMRI) fitting variable that represents the amount of magnetization at thermal equilibrium. M0 maps are RECOMMENDED to use this suffix if generated by qMRI applications (e.g. variable flip angle T1 mapping.                                                                                                                                                                                                                                                                                                                                                                      |
+
+Parametric images listed in the table above are typically generated by
+processing a [file collection](../02-common-principles.html#entity-linked-file-collections).
+Please visit the [file collections appendix](../99-appendices/10-file-collections.html) to see the
+list of suffixes available for quantitative MRI (qMRI) applications associated
+with these maps.
+For any other details on the organization of parametric maps, their
+recommended metadata fields, and the application specific entity or
+metadata requirement levels of [file collections](../99-appendices/10-file-collections.html) that can generate
+them, visit the [qMRI appendix](../99-appendices/11-qmri.html).
+
+### Legacy suffixes (to be deprecated)
```

Comment:
Just "deprecated". We don't have a concept of "legacy" and ultimately chose just to go with deprecation.

Reply:
You can look at how the `phase` suffix is describe under the fMRI section for example text, since this also came up there.

Reply:
19b6211